index.js

```diff
@@ -23,7 +34,7 @@ class Config {
 
     KafkaTopics() {
         var topics = new(Map);
```

Comment:
Should this also have been changed to a let considering we're going to use it in an iteration. I have vague memories of things possibly getting weird when using a var initialized variable in an iteration.